test/testdrive/apd-sum.td

```diff
@@ -0,0 +1,353 @@
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Note that this test uses an append-only source, so doesn't fully express the
+# semantics of summing APD values because values are non-retractable. # A future
```

Comment:
You can use  the ```SELECT * FROM table_with_insertions EXCEPT ALL SELECT * FROM table_with_deletions``` trick to force a retraction to happen. Or open a follow-up ticket for me to improve this test with retractions.